### PR TITLE
location implementation

### PR DESCRIPTION
**Pull request by pmboerth**

### CODE REVIEW

File: backend/src/main/java/com/internally/backend/models/Company.java [hunk 1]
```diff
@@ -22,14 +24,32 @@ public class Company {
     @OneToMany(mappedBy = "company", cascade = CascadeType.ALL)
     private List<Review> reviews = new ArrayList<>();
 
-    protected Company() {}
+    @ManyToMany
+    @JoinTable(
+            name = "company_locations",
+            joinColumns = @JoinColumn(name = "company_id"),
+            inverseJoinColumns = @JoinColumn(name = "location_id"))
+    private Set<Location> companyLocations = new HashSet<>();
```
Comment (aaronkim218) [inline backend/src/main/java/com/internally/backend/models/Company.java:32]: might be a nitpick, but do we need to name companyLocations if its part of the company class. bc then why not name "name" as "companyName" and "industry" as "companyIndustry". i would think locations is more consistent

**Review by aaronkim218 — APPROVED** [review]

File: backend/src/main/java/com/internally/backend/models/Company.java (hunk 1 above)
Comment (aaronkim218) [reply backend/src/main/java/com/internally/backend/models/Company.java:32]: @pmboerth look at this and see what you think. im merging this PR now so we can change this in later PR

**Comment by aaronkim218** [comment]
just noticed theres location stuff in this PR. refer to my comment in location PR

File: backend/src/main/java/com/internally/backend/models/Company.java (hunk 1 above)
Comment (pmboerth) [reply backend/src/main/java/com/internally/backend/models/Company.java:32]: I agree and changed the naming in the most recent PR

**Pull request merged**